.github/workflows/pr.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
       - name: Check bundle size
         # this should match the os and version used in the release.yml
-        if: startsWith(matrix.os, 'ubuntu-22.04') && startsWith(matrix.node, '14.')
+        if: startsWith(matrix.os, 'ubuntu-22.04') && startsWith(matrix.node, '14.') && matrix.node != '14.0.0'
```

Comment:
This stops the check bundle size run from happening twice; we don't want it to run on 14.0.0